gen-bundle-image.sh

```diff
@@ -8,22 +8,18 @@ function set_bundle_variables {
    local version=$1
    local preset=$2
 
-   if [[ ${preset} = "openshift" ]]; then
-       vfkit_bundle_arm64=crc_vfkit_${version}_arm64.crcbundle
-   fi
-   if [[ ${preset} = "podman" ]]; then
-       vfkit_bundle_arm64=crc_podman_vfkit_${version}_arm64.crcbundle
+   local bundlePreset=""
+   if [ ${preset} != 'openshift' ]; then
+       bundlePreset="_${preset}"
    fi
 
-   if [[ ${preset} = "openshift" ]]; then
-       preset=""
-   else
-       preset="_${preset}"
+   if [ ${PRESET} != 'okd' ]; then
+       vfkit_bundle_arm64=crc${bundlePreset}_vfkit_${version}_arm64.crcbundle
    fi
 
-   vfkit_bundle=crc${preset}_vfkit_${version}_amd64.crcbundle
-   libvirt_bundle=crc${preset}_libvirt_${version}_amd64.crcbundle
-   hyperv_bundle=crc${preset}_hyperv_${version}_amd64.crcbundle
+   vfkit_bundle=crc${bundlePreset}_vfkit_${version}_amd64.crcbundle
+   libvirt_bundle=crc${bundlePreset}_libvirt_${version}_amd64.crcbundle
+   hyperv_bundle=crc${bundlePreset}_hyperv_${version}_amd64.crcbundle
```

Comment:
This looks related to supporting `microshift` in addition to `okd`, `podman` and `openshift`...

Reply:
@cfergeau is it still unresolved? or this comment is for commit log change?